### ISSUE TITLE
onPeriodChange not firing initially, firing twice on first period change

### ISSUE THREAD

**Issue by Dylan-Chapman** [richardtallent/vue-simple-calendar#94]
For reference: [https://codepen.io/anon/pen/WgXjgY](https://codepen.io/anon/pen/WgXjgY)

In the pen above, I have a console log firing whenever `onPeriodChange` is fired. On page load, you can see that `onPeriodChange` is not fired. However, if you click the arrows to change the month, you can see the first time the month is changed, `onPeriodChange` is fired twice. I'm assuming these two problems are related in some way. 

As a side note, `onPeriodChange` is a really unfortunate prop name (I read the changelog and understand why you went this route). I primarily use JSX in my application, and when you type a prop that starts with `on`, JSX treats that as an event instead of a prop. I have to work around this by manually clarifying that it's an attribute on the CalendarView tag:

```jsx
{...{
   attrs: {
     onPeriodChange: data => {
        // fetch events
     }
  }
}}
```

**Comment by Dylan-Chapman** [comment]
Yep, something like that would work fine. Vuetify used to use ``{affix}-icon-cb` props for callbacks when clicking on icons (they've since switched to events).

**Comment by richardtallent** [comment]
I believe this has been fixed in the latest version, 4.1.0... at least, my sample app is showing that the function is being called on initialization. The function prop has also been renamed "periodChangedCallback" to work around the JSX issue. Thanks!
